test/edge/Evalue.h

```diff
@@ -0,0 +1,472 @@
+#pragma once
+
+#include <ATen/ATen.h>
+
+namespace torch {
+namespace executor {
```

Comment:
So this is duplicated with fbcode-internal Executorch, is that right? Can we have instructions for (1) if you need to make changes to this file, what needs to happen on fbcode style and (2) under what circumstances can we delete this file and reunify them again?

Reply:
I guess I'm not going to review this for functionality

Reply:
Good point!